internal/app/cloudinfo/api/handlers.go

```diff
@@ -77,12 +76,12 @@ func (r *RouteHandler) getProvider() gin.HandlerFunc {
 	return func(c *gin.Context) {
 		pathParams := GetProviderPathParams{}
 		if err := mapstructure.Decode(getPathParamMap(c), &pathParams); err != nil {
-			r.errorResponder.Respond(c, emperror.With(err, "validation"))
+			r.errorResponder.Respond(c, errors.WithDetails(err, "validation"))
```

Comment:
With details is supposed to be called with key-value pairs: `errors.WithDetails(err, "key", "value")`

Reply:
thanks, will fix in a new PR